velox/substrait/tests/VeloxSubstraitRoundTripPlanConverterTest.cpp

```diff
@@ -286,6 +286,33 @@ TEST_F(VeloxSubstraitRoundTripPlanConverterTest, ifThen) {
   assertPlanConversion(plan, "SELECT if (c0=1, c0 + 1, c1 + 2) as x FROM tmp");
 }
 
+TEST_F(VeloxSubstraitRoundTripPlanConverterTest, NotNullLiteral) {
```

Comment:
naming: NotNullLiteral -> notNullLiteral

VeloxSubstraitRoundTripPlanConverterTest name is too long. It would be nice to shorten it to VeloxSubstraitRoundTripTest in a separate PR.

Reply:
It fixed. I also rename VeloxSubstraitRoundTripPlanConverterTest to VeloxSubstraitRoundTripTest in this PR. Because I think the change is not very big.